packages/react-search-ui-views/src/__tests__/__snapshots__/Result.test.js.snap

```diff
@@ -90,6 +90,60 @@ exports[`renders correctly when there is a URL 1`] = `
 </li>
 `;
 
+exports[`renders correctly when there is a malicious URL 1`] = `
+<li
+  className="sui-result"
+>
+  <div
+    className="sui-result__header"
+  />
+  <div
+    className="sui-result__body"
+  >
+    <ul
+      className="sui-result__details"
+    >
+      <li
+        key="field"
+      >
+        <span
+          className="sui-result__key"
+        >
+          field
+        </span>
+         
+        <span
+          className="sui-result__value"
+          dangerouslySetInnerHTML={
+            Object {
+              "__html": "value",
+            }
+          }
+        />
+      </li>
+      <li
+        key="url"
+      >
+        <span
+          className="sui-result__key"
+        >
+          url
+        </span>
+         
+        <span
+          className="sui-result__value"
+          dangerouslySetInnerHTML={
+            Object {
+              "__html": "javascript://test%0aalert(document.domain)",
+            }
+          }
```

Comment:
Jest snapshots never fail to confuse me sometimes. no idea if it's just me, but `dangerouslySetInnerHTML` here seems to be fairly counterintuitive, shouldn't this just be plain text within a span? 🤷 

that being said if this is of course auto generated code, nothing to do here, just me yelling at clouds

Reply:
@constancecchen After running our new code, `url` will be empty if a malicious url is attempted. 

In that case, this line of code runs: https://github.com/elastic/search-ui/blob/master/packages/react-search-ui-views/src/Result.js#L67

That line of code prints the title, which may or may not have snippets, so it uses `dangerouslySetInnerHTML`.

The title is html escaped [here](https://github.com/elastic/search-ui/blob/master/packages/react-search-ui-views/src/Result.js#L61), which should make that safe.